src/worker.js

```diff
@@ -111,7 +66,10 @@ module.exports = async () => {
         messages: report.results.length ? report.results[0].messages : []
       }
       if (sendRules) {
-        response.fixableRules = Array.from(fixableRules.keys())
+        // You can't emit Maps, convert to Array of Arrays to send back.
+        response.updatedRules = []
+        rulesMetadata.forEach((props, rule) =>
+          response.updatedRules.push([rule, props]))
```

Comment:
Doesn't `Array.from(rulesMetadata)` do what you want here?  And if not, can we use [`rulesMetadata.entries()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/entries)?

Reply:
Honestly I didn't even think to try it. `Array.from(<some Map>)` does _exactly_ this. 👍